helpers.js

```diff
@@ -492,6 +492,17 @@ const getArmletClient = (ethAddress, password, clientToolName = 'truffle') => {
  * @param {Object} config - truffle configuration object.
  */
 async function analyze(config) {
+
+    try {
+        const projectConfig = require([config.working_directory, 'truffle-security.json'].join ('/'));
+        // command line options should overwrite project-level config
+        config = Object.assign(projectConfig, config);
```

Comment:
After switching `projectConfig` and `config`, this is not successfully assigning the config for me, and an exception is being raised at this line (then caught and thought to be a missing file). Could you see what is going on?

Reply:
@nbanmp my best guess would be object.assign may not be able to copy appropriately all the deep, recursive, nested fields of `config` here (which is Truffle's config object). In any event, I just handrolled a check to copy the project level config to to this giant object, only if the key is not already present. So that should work, but please let me know if you find otherwise!